src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderDescriptionController.cs

```diff
@@ -28,11 +29,16 @@ public OrderDescriptionController(IOrderRepository orderRepository)
         public async Task<ActionResult> GetAsync([FromRoute][Required] string orderId)
         {
             var order = await _orderRepository.GetOrderByIdAsync(orderId);
+            var primaryOrganisationId = User.GetPrimaryOrganisationId();

```

Comment:
Only need to get the primary organisation ID if an order is returned. Could this be moved closer to its usage i,e, Just before the line `if (primaryOrganisationId != order.OrganisationId.ToString())`? I think this helps readability keeping context close together.